packages/load-script/src/dom-operations.js

```diff
@@ -3,14 +3,25 @@ import { handleRequestError, handleRequestSuccess } from './callback-handler';
 
 const debug = debugFactory( 'lib/load-script/dom-operations' );
 
-export function createScriptElement( url ) {
+export function createScriptElement( url, args ) {
 	debug( `Creating script element for "${ url }"` );
 	const script = document.createElement( 'script' );
+
 	script.src = url;
 	script.type = 'text/javascript';
-	script.async = true;
 	script.onload = handleRequestSuccess;
 	script.onerror = handleRequestError;
+
+	// Only set the default async attribute if it's not already set in the args,
+	// otherwise in cannot be "unset".
+	if ( ! args || ( args && ! ( 'async' in args ) ) ) {
+		script.async = true;
+	}
```

Comment:
I might be misunderstanding, but if we do the attribute (un)assignment as we do below on L22, this could actually be removed, and `script.async = true` could be set above like we used to do it on L11 before. Thoughts?

Reply:
Hrm, correct. It's simpler so I'll revert to that.